### ISSUE TITLE
How to change First day of the week to Monday

### ISSUE THREAD

**Issue by MariuszGSL** [flavienlaurent/datetimepicker#8]
I have successfully implemented your library in my application. 
Thank you for this great calendar widget.

I have one major issue with your current implementation:
it displays the month calendar with the first column set to Sunday.
This is only applicable to North America and maybe a few other countries.
The rest, vast majority of the world countries start their calendars on Monday. 

How to change your library code to show the month calendar starting each week on Monday, not Sunday ?

As an example, the Samsung S Planner app allows users to change the First day of the week to their preferred day, either: Saturday, Sunday or Monday.

**Comment by gabrielittner** [comment]
The first day of week is determined by your locale. Set it to something like English (UK) or German and you will have Monday as first day of week.

**Comment by MariuszGSL** [comment]
My phone (Samsung Galaxy Note 2) was already set to the correct locale (English New Zealand). I have also tried English (Australia) with the same effect: the calendar week still starts on Sunday.
I then downloaded the app "Locale Select" and changed it to English (Belgium) and got it working !
I wonder why the English NZ and Australia locales do not work ?  We certainly use our calendars here with Monday as the first day of the week (unlike in the US).
Is there another way to force this calendar date picker to permanently show the weeks starting on Monday ?
For example, the Samsung S Planner app has an internal setup switch to change the First day of the week setting for its calendar. Most likely all local (NZ/Australia) users will have the same problem with the default locale.

P.S. Thank you Gabriel for this reply. I was inspired by your beautiful and clean Timetable app and chose this particular date picker for my application.
By the way: Your Timetable application date picker has the same locale problem.
